### PR TITLE
update gpg keys

### PR DESCRIPTION
**Pull request by MartinNowak**
- add @wilzbach
- add nightly build bot
- update public key signatures (expiration dates and ids)
- sign updated keyring

### CODE REVIEW

File: gpg_keys.dd [hunk 1]
```diff
@@ -3,16 +3,26 @@ Ddoc
...
 
-pub   4096R/0xC13C70B87267B52D 2012-03-20 [expires: 2016-03-19]
-      Key fingerprint = 0B96 609E 270F 565C 1329  2B24 C13C 70B8 7267 B52D
-uid                            David Soria Parra $(LT)dsp@php.net$(GT)
-sub   4096R/0x4A208C733953AF82 2012-03-20 [expires: 2016-03-19]
+pub   4096R/0x2055F76601A36FB0 2015-11-24 [expires: 2020-11-22]
+      Key fingerprint = 8FDB 8D35 7AF4 68A9 428A  CE3C 2055 F766 01A3 6FB0
+uid                            Sebastian Wilzbach $(LT)sebi@wilzbach.me$(GT)
+uid                            Sebastian Wilzbach $(LT)seb@wilzba.ch$(GT)
+sub   4096R/0xFEB1F8D40328010D 2015-11-24 [expires: 2020-11-22]
```
Comment (wilzbach) [inline gpg_keys.dd]: FYI: I updated the expiration of my key a few days ago as I didn't want to experience the same breakage as you did.
So while you are at updating our keyring, you might use the fingerprint too:

http://pgp.mit.edu/pks/lookup?op=vindex&search=0x2055F76601A36FB0
https://pgp.wilzba.ch/seb.asc

**Comment by wilzbach** [comment]
Not sure why this is failing on the server, it's passing locally. Trying a restart.

**Comment by MartinNowak** [comment]
Done @wilzbach 

Not sure about GDC and LDC, let's see if there is interest and who would be the right person to add (@jpf91, @kinke), atm. none of those releases/tags are signed.

**Comment by wilzbach** [comment]
> Done @wilzbach

Did you forget to push? I don't see @CyberShadow's key in the keyring (nor the updated fingerprint of mine).

**Comment by MartinNowak** [comment]
> Did you forget to push? I don't see @CyberShadow's key in the keyring (nor the updated fingerprint of mine).

Indeed @wilzbach, sorry.

**Pull request merged**